src/help.json

```diff
@@ -32,5 +32,10 @@
     "role": "@everyone",
     "description":
       "[For testing purposes] Check if you are in our database and adds you if you're not"
+  },
+  {
+    "names": "!emoji",
+    "role": "@everyone",
```

Comment:
Not `@everyone` can add emojis, only people with the `Camper` role and above.

Reply:
Sad